stix_shifter_modules/sysdig/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,356 @@
+        "object": "finding"
+      },
+      "proc.cmdline": {
+        "key": "process.command_line",
+        "object": "proc"
+      },
+      "proc.name": [
+        {
+          "key": "file.name",
+          "object": "file"
+        },
+        {
+          "key": "process.name",
+          "object": "proc"
+        },
+        {
+          "key": "process.binary_ref",
+          "object": "proc",
+          "references": "file"
+        }
+      ],
+      "proc.pid": {
+        "key": "process.pid",
+        "object": "proc",
+        "transformer": "ToInteger"
+      },
+      "proc.sid": {
+        "key": "process.x_sid",
+        "object": "proc"
+      },
+      "proc.exepath": [
+        {
+          "key": "directory.path",
+          "object": "file_dir"
+        },
+        {
+          "key": "file.parent_directory_ref",
+          "object": "file",
+          "references": "file_dir"
+        }
+      ],
+      "proc.cwd": {
+        "key": "process.cwd",
+        "object": "proc"
+      },
+      "proc.pcmdline": {
+        "key": "process.x_pcmdline",
+        "object": "proc"
+      },
+      "proc.pname": [
+        {
+          "key": "file.name",
+          "object": "parent_file"
+        },
+        {
+          "key": "process.name",
+          "object": "parent_proc"
+        },
+        {
+          "key": "process.parent_ref",
+          "object": "proc",
+          "references": "parent_proc"
+        },
+        {
+          "key": "process.binary_ref",
+          "object": "parent_proc",
+          "references": "parent_file"
+        }
+      ],
+      "proc.ppid": {
+        "key": "process.pid",
+        "object": "parent_proc",
+        "transformer": "ToInteger"
+      },
+      "user.loginname": {
+        "key": "user-account.account_login",
+        "object": "user"
+      },
+      "user.loginuid": {
+        "key": "user-account.x_loginuid",
+        "object": "user"
+      },
+      "user.name": [
+        {
+          "key": "user-account.display_name",
+          "object": "user"
+        },
+        {
+          "key": "process.creator_user_ref",
+          "object": "proc",
+          "references": "user"
+        }
+      ],
+      "user.uid": {
+        "key": "user-account.user_id",
+        "object": "user"
+      },
+      "proc.aname[1]": {
+        "key": "process.x_ancestor_name_1",
+        "object": "proc"
+      },
+      "proc.aname[2]": {
+        "key": "process.x_ancestor_name_2",
+        "object": "proc"
+      },
+      "proc.aname[3]": {
+        "key": "process.x_ancestor_name_3",
+        "object": "proc"
+      },
+      "proc.aname[4]": {
+        "key": "process.x_ancestor_name_4",
+        "object": "proc"
+      },
+      "proc.aname[5]": {
+        "key": "process.x_ancestor_name_5",
+        "object": "proc"
+      },
+      "proc.aname[6]": {
+        "key": "process.x_ancestor_name_6",
+        "object": "proc"
+      },
+      "proc.aname[7]": {
```

Comment:
where does the number in `proc.aname[7]` field come from? Is it always up to 7?

Reply:
For most of the events, proc.aname is coming up to 3 or 4 only, we have seen maximum it coming up to 7 for few events . Hence we are supporting up to 7 for this field proc.aname.

Reply:
proc.aname values (ancestor names) are combined as list and moved to x_parent_names under parent process.